stetho/src/main/java/com/facebook/stetho/inspector/elements/NodeDescriptor.java

```diff
@@ -33,4 +33,6 @@
   void getAttributes(Object element, AttributeAccumulator attributes);
 
   void setAttributesAsText(Object element, String text);
+
+  void getStyleProperties(Object element, Accumulator<StyleInfo> accumulator);
```

Comment:
we have a naming problem here ... are we getting the element's style *properties* or *info* ? we should choose one word and stick with it everywhere (in other words, consistency please!)